### ISSUE TITLE
segmentation fault `source kvm.sh` OSX (possibly zsh)

### ISSUE THREAD

**Issue by alexgorbatchev** [aspnet/Home#166]
On OSX with ZSH

```bash
$ zsh
$ source kvm.sh
[1]    84682 segmentation fault  zsh
```

However the following works fine

```bash
$ bash
$ source kvm.sh
$ kvm
K Runtime Environment Version Manager - Build 10017

USAGE: kvm <command> [options]
```

**Comment by Praburaj** [comment]
@natemcmaster this looks like a dup of https://github.com/aspnet/kvm/issues/83. Home repo now `kvm` scripts contains all the fixes. Could you try updating the kvm script (I think you could reinstall the brew formula to get the latest).

**Comment by natemcmaster** [comment]
@Praburaj appears to be fixed, at least OS X 10.9.5 :smiley:

**Comment by Praburaj** [comment]
Great. Thanks @natemcmaster for confirming :smiley:
